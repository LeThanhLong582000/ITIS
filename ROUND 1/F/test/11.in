346104637 8