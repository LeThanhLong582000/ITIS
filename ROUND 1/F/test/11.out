34610463700000000